src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -285,6 +285,8 @@ internal static int CombineHashCodes(int h1, int h2, int h3, int h4, int h5, int
     public struct ValueTuple<T1>
         : IEquatable<ValueTuple<T1>>, IStructuralEquatable, IStructuralComparable, IComparable, IComparable<ValueTuple<T1>>, ITupleInternal
     {
+        private static readonly IEqualityComparer<T1> T1Comparer = EqualityComparer<T1>.Default;
```

Comment:
Does it make a measurable improvement to type these static fields as `EqualityComparer<T>` instead of the `IEqualityComparer<T>` interface (to avoid interface dispatch)?